packages/palette-docs/gatsby-config.js

```diff
@@ -58,6 +58,12 @@ module.exports = {
         manualInit: true,
       },
     },
+    {
+      resolve: `gatsby-plugin-s3`,
+      options: {
+        bucketName: "artsy-palette",
```

Comment:
What I've previously seen for S3 based hosting is that [the bucket name has to match the hostname](https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html#VirtualHostingCustomURLs) of the site being served — so we might need `palette.artsy.net` as the name of the bucket.

On the other hand, I'm not sure if that's required if we make a CloudFront distribution out of this bucket — something we might consider in order to get features like https, caching etc.

Reply:
Yup, we'll be putting this behind cloudfront 👍